test/test_torch.py

```diff
@@ -14625,6 +14625,7 @@ def test_remainder_edge_cases(self, device, dtype):
         self.assertEqual(r.dtype, a.dtype)
 
     @slowTest
+    @onlyOnCPUAndCUDA
```

Comment:
Are there any other devices besides CPU, CUDA, or XLA that we target? Because if there are, this would prevent this test from running on those devices as well.

Reply:
I don't believe so at the moment. I had the same concern as you, but according to git history "@onlyOnCPUAndCUDA" was explicitly created in order to skip tests on XLA in particular.